src/components/coin-icon/CoinIcon.js

```diff
@@ -34,21 +36,31 @@ const CoinIcon = ({
 
   return (
     <View>
-      <StyledCoinIcon
-        {...props}
-        address={props.mainnet_address || address}
-        color={color}
-        fallbackRenderer={CoinIconFallback}
-        forceFallback={forceFallback}
-        // force update on change symbol due to ImageCache strategy
-        key={symbol}
-        shadowColor={
-          forcedShadowColor ||
-          (isDarkMode ? colors.shadow : tokenMetadata?.shadowColor || color)
-        }
-        size={size}
-        symbol={symbol}
-      />
+      {symbol !== 'contract' ? (
```

Comment:
I'll probably create a constant for this, didn't like to add a check here, but I didn't want to add the contract image to that list (`react-coin-icon`)

Reply:
hmm @mikedemarais @brunobar79